horace_core/configuration/@hor_config/hor_config.m

```diff
@@ -296,14 +296,20 @@
             if use
                 % Configure mex usage
                 % --------------------
-                [~, n_errors, can_combine_with_mex] = check_horace_mex();
+                [~, n_errors, can_combine_with_mex,can_use_herbert_mpi] = check_horace_mex();
```

Comment:
as above unclear why herbert is appearing here

Reply:
may be gave this variable better name